corehq/apps/userreports/reports/filters.py

```diff
@@ -54,23 +67,28 @@ def to_sql_values(self):
 
 
 class NumericFilterValue(FilterValue):
+    operators_to_filters = {
+        '=': EQFilter,
+        '!=': NOTEQFilter,
+        '>=': GTEFilter,
+        '>': GTFilter,
+        '<=': LTEFilter,
+        '<': LTFilter,
+    }
 
     def __init__(self, filter, value):
         assert filter.type == "numeric"
         assert (isinstance(value, dict) and "operator" in value and "operand" in value) or value is None
         if value:
-            assert value['operator'] in ["=", "!=", "<", "<=", ">", ">="]
+            assert value['operator'] in self.operators_to_filters.keys()
```

Comment:
You need the `.keys()`. You can just say `assert value['operator'] in self.operators_to_filters`